### PR TITLE
Moes ZLD-RCW and Miboxer FUT039Z LED Controller

### PR DESCRIPTION
**Pull request by kirovilya**

### CODE REVIEW

File: converters/toZigbee.js [hunk 1]
```diff
@@ -5368,6 +5368,23 @@ const converters = {
...
     },
+    tuya_do_not_disturb: {
+        key: ['do_not_disturb'],
+        convertSet: async (entity, key, value, meta) => {
+            await entity.command('lightingColorCtrl', 'tuyaDoNotDisturb', {enable: value ? 1 : 0});
+            return {state: {do_not_disturb: value}};
+        },
+    },
+    tuya_color_power_on_behavior: {
+        key: ['tuya_color_power_on_behavior'],
```
Comment (Koenkk) [inline converters/toZigbee.js:5379]: Having `color_power_on_behavior` sounds better in my opinion.

File: converters/toZigbee.js [hunk 2]
```diff
@@ -5368,6 +5368,23 @@ const converters = {
...
+        convertSet: async (entity, key, value, meta) => {
+            const lookup = {'initial': 0, 'previous': 1, 'cutomized': 2};
+            utils.validateValue(value, Object.keys(lookup));
+            await entity.command('lightingColorCtrl', 'tuyaOnStartUp', {
+                mode: lookup[value]*256, data: [0, 0, 0, 0, 0, 0, 0, 0, 0, 0]});
+            return {state: {power_on_behavior: value}};
```
Comment (Koenkk) [inline converters/toZigbee.js:5385]: The returned property here should match the key, (e.g. `return {state: {color_power_on_behavior: value}};`

File: converters/toZigbee.js (hunk 1 above)
Comment (sjorge) [reply converters/toZigbee.js:5379]: Could we have this under `color_options` ? https://github.com/Koenkk/zigbee-herdsman-converters/blob/1c60448990c69d0737268381fcac6e8ce5253748/converters/toZigbee.js#L100 we use that for some of the generic lightingColorCtrl options, so having it use the same key would be more consistant?

Comment (kirovilya) [reply converters/toZigbee.js:5379]: @sjorge look at https://github.com/Koenkk/zigbee-herdsman/pull/390
it is not common case - it it tuya commands

File: converters/toZigbee.js (hunk 2 above)
Comment (kirovilya) [reply converters/toZigbee.js]: renamed

File: converters/toZigbee.js (hunk 1 above)
Comment (Koenkk) [reply converters/toZigbee.js]: Both are good, since I don't expect any other options this is fine.

**Pull request merged**